docs/source/concepts.rst

```diff
@@ -241,9 +241,15 @@ and be registered with :meth:`~ignite.engine.engine.Engine.register_events` in a
         CUSTOM_STARTED = 'custom_started'
         CUSTOM_COMPLETED = 'custom_completed'
 
-    engine.register_events(*CustomEvents)
+    engine.register_events(
+        *CustomEvents,
+        event_to_attr={
```

Comment:
This part is not required actually. I'd put everything in the following way:
- simple custom events without filtering (current doc)
- custom events with filtering:
   - engine.state should have corresponding attributes
   - event_to_attr is required when `register_events`
   - should increase counters (subject to be removed)